packages/data/src/world/mq/water_temple_mq.yml

```diff
@@ -13,27 +13,28 @@
-    "MQ Water Temple GS Lizalfos Hallway": "event(WATER_LEVEL_LOW) && can_use_din"
-    "MQ Water Temple GS High Water Changer": "event(WATER_LEVEL_LOW) && can_longshot"
+    "MQ Water Temple Central Pillar Chest": "can_play_time && can_use_din && can_hookshot && has_iron_boots && has_tunic_zora_strict && event(WATER_LEVEL_LOW)"
+    "MQ Water Temple GS Lizalfos Hallway": "event(WATER_LEVEL_LOW) && can_use_din && (has_hover_boots || can_hookshot || can_play_elegy)"
+    "MQ Water Temple GS High Water Changer": "event(WATER_LEVEL_MID) && can_longshot"
     "MQ Water Temple Pot Ruto 1": "has_iron_boots && has_tunic_zora && (can_hookshot || event(WATER_LEVEL_LOW))"
     "MQ Water Temple Pot Ruto 2": "has_iron_boots && has_tunic_zora && (can_hookshot || event(WATER_LEVEL_LOW))"
     "MQ Water Temple Pot Storage Room 1": "has_iron_boots && has_tunic_zora && can_hookshot"
     "MQ Water Temple Pot Storage Room 2": "has_iron_boots && has_tunic_zora && can_hookshot"
     "MQ Water Temple Pot Storage Room 3": "has_iron_boots && has_tunic_zora && can_hookshot"
-    "MQ Water Temple Pot Twisted Room Entrance": "has_iron_boots && has_tunic_zora && (can_hookshot || (event(WATER_LEVEL_LOW) && has_hover_boots))"
-    "MQ Water Temple Pot Twisted Room Right 1": "has_iron_boots && has_tunic_zora && (can_hookshot || (event(WATER_LEVEL_LOW) && has_hover_boots))"
-    "MQ Water Temple Pot Twisted Room Right 2": "has_iron_boots && has_tunic_zora && (can_hookshot || (event(WATER_LEVEL_LOW) && has_hover_boots))"
-    "MQ Water Temple Pot Twisted Room Cage 1": "event(WATER_LEVEL_LOW) && can_use_din"
-    "MQ Water Temple Pot Twisted Room Cage 2": "event(WATER_LEVEL_LOW) && can_use_din"
-    "MQ Water Temple Pot Room Before High Water 1": "event(WATER_LEVEL_LOW) && can_hookshot"
-    "MQ Water Temple Pot Room Before High Water 2": "event(WATER_LEVEL_LOW) && can_hookshot"
-    "MQ Water Temple Pot Room Before High Water 3": "event(WATER_LEVEL_LOW) && can_hookshot"
+    "MQ Water Temple Pot Twisted Room Entrance": "has_iron_boots && has_tunic_zora && (can_hookshot || (event(WATER_LEVEL_LOW) && has_hover_boots) || can_play_elegy)"
+    "MQ Water Temple Pot Twisted Room Right 1": "has_iron_boots && has_tunic_zora && (can_hookshot || (event(WATER_LEVEL_LOW) && has_hover_boots) || can_play_elegy)"
+    "MQ Water Temple Pot Twisted Room Right 2": "has_iron_boots && has_tunic_zora && (can_hookshot || (event(WATER_LEVEL_LOW) && has_hover_boots) || can_play_elegy)"
+    "MQ Water Temple Pot Twisted Room Cage 1": "event(WATER_LEVEL_LOW) && can_use_din && (has_hover_boots || can_hookshot || can_play_elegy)"
+    "MQ Water Temple Pot Twisted Room Cage 2": "event(WATER_LEVEL_LOW) && can_use_din && (has_hover_boots || can_hookshot || can_play_elegy)"
+    "MQ Water Temple Pot Room Before High Water 1": "event(WATER_LEVEL_MID)"
+    "MQ Water Temple Pot Room Before High Water 2": "event(WATER_LEVEL_MID)"
+    "MQ Water Temple Pot Room Before High Water 3": "event(WATER_LEVEL_MID)"
```

Comment:
these three should be reverted with the addition of elegy as an alternative to hook.

Reply:
So they should be `can_hookshot || can_play_elegy`?